Cargo.lock

```diff
@@ -3235,6 +3327,30 @@ version = "0.2.68"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1d649a3145108d7d3fbcde896a468d1bd636791823c9921135218ad89be08307"
 
+[[package]]
+name = "wasm-bindgen-test"
+version = "0.3.18"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "34d1cdc8b98a557f24733d50a1199c4b0635e465eecba9c45b214544da197f64"
+dependencies = [
+ "console_error_panic_hook",
+ "js-sys",
+ "scoped-tls",
+ "wasm-bindgen",
+ "wasm-bindgen-futures",
+ "wasm-bindgen-test-macro",
+]
+
+[[package]]
+name = "wasm-bindgen-test-macro"
+version = "0.3.18"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e8fb9c67be7439ee8ab1b7db502a49c05e51e2835b66796c705134d9b8e1a585"
+dependencies = [
+ "proc-macro2 1.0.24",
+ "quote 1.0.7",
+]
```

Comment:
Where do these deps come from?

Reply:
Quick grep of the Cargo.lock file, they're from reqwest

Reply:
Maybe that's related to https://github.com/seanmonstar/reqwest/pull/1093

Reply:
It's strange that `reqwest` gets upgraded anyway. Is that intentional?

Reply:
Not really, just merged with master and ran cargo update 🤔

Reply:
> Not really, just merged with master and ran cargo update 🤔

Bad idea 😅  I suggest upgrading manually only crates you're interested in. @caspervonb could you revert non-rustyline related crate updates?